lib/OpenQA/WebAPI/Controller/Appearance.pm

```diff
@@ -0,0 +1,23 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::WebAPI::Controller::Appearance;
+use Mojo::Base 'Mojolicious::Controller', -signatures;
+
+sub index ($self) {
+    $self->render;
+}
```

Comment:
Is a function that only calls `$self->render` actually required?

Reply:
No, but it's the "correct" way to deal with it. I'm moving away from relying on auto-rendering, since it's a feature Mojolicious is soft-deprecating with the introduction of mojo.js.